src/hooks/transactions/useTransactionsTracker/useUpdateTrackedTransactions.ts

```diff
@@ -10,23 +10,24 @@ import {
 } from 'types';
 import { refreshAccount } from 'utils/account/refreshAccount';
 
-export function useUpdateBatch() {
+// TODO: This function seems duplicate to the manageTransaction from checkBatch.ts file
```

Comment:
is it still used? why not combine them into one?

Reply:
This will be changed in the new sdk-dapp-core. why work on it twice?